common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -128,20 +129,24 @@ void extractState(@NonNull final Bundle state) {
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         final String methodName = "#onCreateView";
         final View view = inflater.inflate(R.layout.common_activity_authentication, container, false);
+        final String[] javascriptToExecute = new String[1];
         mProgressBar = view.findViewById(R.id.common_auth_webview_progressbar);
 
         final AzureActiveDirectoryWebViewClient webViewClient = new AzureActiveDirectoryWebViewClient(
                 getActivity(),
                 new AuthorizationCompletionCallback(),
                 new OnPageLoadedCallback() {
                     @Override
-                    public void onPageLoaded() {
+                    public void onPageLoaded(final String url) {
                         mProgressBar.setVisibility(View.INVISIBLE);
-                    }
-                },
-                new OnPageCommitVisibleCallback() {
-                    @Override
-                    public void onPageCommitVisible() {
+                        try {
+                            javascriptToExecute[0] = String.format("window.expectedUrl = '%s';%n%s",
```

Comment:
>javascriptToExecute[0] [](http://example.com/codeflow?start=28&length=22)

This is never used. Shouldn't this be used insted of mPostPageLoadedJavascript in the code below?
Also why is it an array?

Reply:
Modified code to use javascriptToExecute[0] instead of mPostPageLoadedJavascript.
As javascriptToExecute[0] needs to be declared as final, so declared it as 1 element array.